src/emcpy/plots/create_plots.py

```diff
@@ -1,14 +1,150 @@
 import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
 import matplotlib.pyplot as plt
+from matplotlib import rcParams
 from matplotlib.colors import Normalize
 from scipy.interpolate import interpn
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+from emcpy.plots.map_tools import Domain, MapProjection
 from emcpy.stats import get_linear_regression
 
+__all__ = ['EMCPyPlots', 'CreatePlot', 'CreateMap']
```

Comment:
Would a user ever call the base class? If not, should `EMCPyPlots` be removed here?